src/components/Dialog.tsx

```diff
@@ -136,7 +136,7 @@ export default class Dialog extends Component<DialogProps, DialogState> {
           </DropdownList>
         </Dropdown>
         <br />
-        <Button onClick={() => this.props.sdk.close('Done!')}>Done</Button>
+        <Button onClick={() => this.props.sdk.close(undefined)}>Done</Button>
```

Comment:
nit: is there a reason why we're explicitly passing in undefined here?

Reply:
So this callback will exit in the [Field component](https://github.com/imgix/contentful/blob/main/src/components/Field.tsx#L50-53), whatever gets passed through `field.setValue()` will be returned by the Contentful API. I wanted to be explicit so some other empty value like `''` doesn't get returned, but you make a good point; there might not be a reason to explicitly pass it here. I'll make a note to test both and see if there's any difference end-to-end.